tests/test_report.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from . import v
+assert v  # Silence pyflakes
+
+mock_code = """\
+import foo
+
+class Foo:
+    '''
+    This is a long class
+    '''
+    def __init__(self):
+        print("Initialized foo")
+
+    def bar(self):
+        self.foobar = "unused attribute"
+        foobar = "unused variable"
+        return
+        print("A small unused function")
+
+    @property
+    def myprop(self):
+        pass
+"""
+
+
+@pytest.fixture
+def check_report(v, capsys, tmpdir):
+    def test_report(code, expected, min_confidence=0, sort_by_size=False,
```

Comment:
We don't need to test the sorting here. Also, we don't need temporary files, if we use scan instead of scavenge.

Reply:
I wrote this test for ensuring that `size_report` stays intact, but I guess it is better off without at test 😛